packages/wonder-blocks-dropdown/src/components/__tests__/action-item.test.js

```diff
@@ -1,33 +1,53 @@
+// @flow
 import * as React from "react";
-import renderer from "react-test-renderer";
 import {MemoryRouter, Link} from "react-router-dom";
 import {mount} from "enzyme";
-import "jest-enzyme";
+import {render, screen} from "@testing-library/react";
 
 import ActionItem from "../action-item.js";
 
 describe("ActionItem", () => {
     it("should render with disabled styles", () => {
-        const actionItem = renderer.create(
-            <ActionItem href="/foo" disabled={true} />,
-        );
+        // Arrange
+
+        // Act
+        render(<ActionItem href="/foo" label="Example" disabled={true} />);
 
-        expect(actionItem).toMatchSnapshot();
+        // Assert
+        expect(screen.getByRole("menuitem")).toBeDisabled();
     });
 
     it("should render an anchor if there's no router", () => {
-        const wrapper = mount(<ActionItem href="/foo" />);
+        // Arrange
 
+        // Act
+        const wrapper = mount(<ActionItem href="/foo" label="Example" />);
```

Comment:
Keeping this with Enzyme because I still don't find a clean way to find elements by tagName with RTL.

Reply:
You should be able to use `getByRole("link")` in this situation.  If you really needed to query by tag name you could do `document.querySelectorAll("a")`.

Reply:
The thing is that we are passing down a `role` prop so the component uses `menuitem` by default. That's why I needed to keep using `a` for that case, but the RTL linter complains about that, so we would need to use `// eslint-disable-next-line testing-library/no-node-access` to fix that. I'm wondering if this test becomes redundant with RTL.

Reply:
Having to add `// eslint-disable-next-line testing-library/no-node-access` from time to time doesn't seem that bad.  You could even disable that rule for this entire test file if need be.